4.sc_Morphology_Neural_Network_MLP_Model/3.model_visualization/scripts/multiclass_visualization_call.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=16
+#SBATCH --mem=300G
+#SBATCH --partition=amem
+#SBATCH --qos=mem
+#SBATCH --time=24:00:00
+#SBATCH --output=sample-%j.out
+
+module load anaconda
+
+conda activate Interstellar
+
+cell_types=( SHSY5Y PBMC )
+model_names=( MultiClass_MLP_h202_remove MultiClass_MLP )
+jupyter nbconvert --to=script --FilesWriter.build_directory=. ../notebooks/*.ipynb*
+
+
```

Comment:
consider adding a comment here to provide information what these loops are doing